docs/layers/polygon-layer.md

```diff
@@ -93,6 +93,14 @@ Whether to generate a line wireframe of the hexagon. The outline will have
 
 Requires the `extruded` prop to be true.
 
+##### `elevationScale` (Number, optional)
```

Comment:
How should we document new features that are added in patch releases? Should we add them to the `What's New` section in the next release? Maybe just double check that these new props all mentioned in the 4.2 `What's New` already?